### PR TITLE
FIX / Logout terminates user session and displays login screen

### PR DESCRIPTION
**Pull request by cody-daigle**

### CODE REVIEW

File: server/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import path from 'path';
 import dotEnv from 'dotenv';
-import express, { Request, Response } from 'express';
+import express, { NextFunction, Request, Response } from 'express';
```
Comment (mikesamm) [inline server/index.ts]: Doesn't look like NextFunction is being used at the moment, but maybe intended for line 138?

**Review by mikesamm — APPROVED** [review]

File: client/src/components/Login.tsx [hunk 2]
```diff
@@ -11,21 +14,33 @@ const Login = (): ReactElement => {
...
       minHeight='100vh'
     >
-      <Box>
-        <Typography variant='h2' align='center' gutterBottom>
-          <Link to='/'>
-            <h1>mkDev</h1>
-          </Link>
+      <Box alignContent={'center'}>
+        <Typography
+          variant='h4'
```
Comment (AlexPHebert2000) [inline client/src/components/Login.tsx]: For accessibility reasons, these should be h1s with their font sizes changed,

File: client/src/components/Logout.tsx [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+
+import axios from 'axios';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Login from './Login';
+import { useNavigate } from 'react-router-dom';
 
 const Logout = (): ReactElement => {
+  useEffect(() => {
+    axios.post('/logout').then(() => console.log('Logged out!'));
```
Comment (AlexPHebert2000) [inline client/src/components/Logout.tsx]: Remove console log

**Review by AlexPHebert2000 — CHANGES_REQUESTED** [review]
Remove console log

File: client/src/components/Login.tsx (hunk 2 above)
Comment (cody-daigle) [reply client/src/components/Login.tsx]: Accessibility? Every text we implement needs to be an H1?

**Review by AlexPHebert2000 — APPROVED** [review]

**Pull request merged**